packages/app/src/app/overmind/namespaces/live/actions.ts

```diff
@@ -187,6 +192,57 @@ export const sendCurrentSelection: Action = ({ state, effects }) => {
   }
 };
 
+export const sendCurrentViewRange: Action = ({ state, effects }) => {
+  if (!state.live.roomInfo) {
+    return;
+  }
+
+  if (state.live.isCurrentEditor) {
+    const { liveUserId, currentViewRange } = state.live;
+    if (liveUserId && currentViewRange) {
+      effects.live.sendUserViewRange(
+        state.editor.currentModuleShortid,
+        liveUserId,
+        currentViewRange
+      );
+    }
+  }
```

Comment:
Use early return instead of nested code to make it a bit more clear

```suggestion
  if (!state.live.isCurrentEditor) {
    return;
  }
  
  const { liveUserId, currentViewRange } = state.live;
  if (liveUserId && currentViewRange) {
    effects.live.sendUserViewRange(
      state.editor.currentModuleShortid,
      liveUserId,
      currentViewRange
    );
  }
```

Reply:
Nice!